tsconfig.json

```diff
@@ -4,6 +4,10 @@
     "paths": {
       "@/components/*": ["components/*"]
     },
+    "typeRoots": [
+      "types",
```

Comment:
This is my preference for where to keep them custom third party types, but happy to move them elsewhere if other prefer.